cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-item-returned.cy.js

```diff
@@ -0,0 +1,295 @@
+              },
+            ],
+            items: [
+              {
+                barcode: item.barcode,
+                status: { name: 'Available' },
+                permanentLoanType: { id: testData.loanTypeId },
+                materialType: { id: testData.materialTypeId },
+              },
+            ],
+          }).then((specialInstanceIds) => {
+            itemsData.itemsWithSeparateInstance[index].instanceId = specialInstanceIds.instanceId;
+            itemsData.itemsWithSeparateInstance[index].holdingId = specialInstanceIds.holdingIds[0].id;
+            itemsData.itemsWithSeparateInstance[index].itemId = specialInstanceIds.holdingIds[0].itemIds;
+          });
+        });
+        cy.wrap(itemsData.itemsWithSeparateInstance).as('items');
+      });
+
+    UsersOwners.createViaApi(owner.body).then((response) => {
+      owner.data = response;
+      PaymentMethods.createViaApi(response.id).then(resp => {
+        testData.paymentMethodId = resp.id;
+      });
+    });
+    LostItemFeePolicy.createViaApi(lostItemFeePolicyBody);
+    LoanPolicy.createApi(loanPolicyBody);
+    PatronGroups.createViaApi(patronGroup.name).then((res) => {
+      patronGroup.id = res;
+    });
+    CirculationRules.getViaApi().then((response) => {
+      originalCirculationRules = response.rulesAsText;
+      const ruleProps = CirculationRules.getRuleProps(response.rulesAsText);
+      ruleProps.l = loanPolicyBody.id;
+      ruleProps.i = lostItemFeePolicyBody.id;
+      CirculationRules.addRuleViaApi(originalCirculationRules, ruleProps, 't ', testData.loanTypeId);
+    });
+
+    cy.createTempUser(
+      [
+        permissions.uiUsersSettingsOwners.gui,
+        permissions.uiUsersCreatePatronConditions.gui,
+        permissions.uiUsersCreatePatronLimits.gui,
+        permissions.checkinAll.gui,
+        permissions.checkoutAll.gui,
+        permissions.uiUsersView.gui,
+        permissions.okapiTimersPatch.gui,
+      ],
+      patronGroup.name
+    )
+      .then((userProperties) => {
+        userData.username = userProperties.username;
+        userData.password = userProperties.password;
+        userData.userId = userProperties.userId;
+        userData.barcode = userProperties.barcode;
+        UserEdit.addServicePointViaApi(testData.userServicePoint.id, userData.userId, testData.userServicePoint.id);
+        cy.getToken(userData.username, userData.password);
+        UserLoans.updateTimerForAgedToLost('minute');
+        cy.getAdminToken();
+      })
+      .then(() => {
+        cy.get('@items').each((item) => {
+          Checkout.checkoutItemViaApi({
+            id: uuid(),
+            itemBarcode: item.barcode,
+            loanDate: moment.utc().format(),
+            servicePointId: testData.userServicePoint.id,
+            userBarcode: userData.barcode,
+          });
+        });
+
+        UserLoans.getUserLoansIdApi(userData.userId).then((response) => {
+          const resp = response.loans;
+          cy.log(resp);
```

Comment:
why do you need cy.log in the pull request?

Reply:
done